README.md

```diff
@@ -35,6 +35,26 @@ Read the [Roadmap](./ROADMAP.md).
 
 ## Quick start
 
+
+#### Run tidb with docker
+
+You can quickly test tidb with docker, the source repository contains Dockerfile,
+You can build docker and then run a docker container with tidb.
+To install docker on your system, you can read the document on https://docs.docker.com/
+
+
+```
+git clone https://github.com/pingcap/tidb.git
+cd tidb
+docker build --rm -t tidb-server .
+docker images
+docker run -d -p 4000:4000 --name tidb-server tidb-server
+```
+Then you can connect 127.0.0.1:4000 by mysql client or telnet.
+The tidb is ready for serving.
+
+Please do not just put this on production environment, you need to write your own config to replace the default config.
```

Comment:
we did have some config 
```
	store     = flag.String("store", "goleveldb", "registered store name, [memory, goleveldb, boltdb]")
	storePath = flag.String("path", "/tmp/tidb", "tidb storage path")
	logLevel  = flag.String("L", "debug", "log level: info, debug, warn, error, fatal")
	port      = flag.String("P", "4000", "mp server port")
```
these config should be modify if we need to deploy to production server